tools/gfx/vulkan/render-vk.cpp

```diff
@@ -8220,6 +8223,118 @@ Result VKDevice::createMutableRootShaderObject(
     return SLANG_OK;
 }
 
+VkSampleCountFlagBits translateSampleCount(uint32_t sampleCount)
+{
+    switch (sampleCount)
+    {
+    case 1:        return VK_SAMPLE_COUNT_1_BIT;
+    case 2:        return VK_SAMPLE_COUNT_2_BIT;
+    case 4:        return VK_SAMPLE_COUNT_4_BIT;
+    case 8:        return VK_SAMPLE_COUNT_8_BIT;
+    case 16:       return VK_SAMPLE_COUNT_16_BIT;
+    case 32:       return VK_SAMPLE_COUNT_32_BIT;
+    case 64:       return VK_SAMPLE_COUNT_64_BIT;
+    default:
+        assert(!"Unsupported sample count");
+        return VK_SAMPLE_COUNT_1_BIT;
+    }
+}
+
+VkCullModeFlags translateCullMode(CullMode cullMode)
+{
+    switch (cullMode)
+    {
+    case CullMode::None:        return VK_CULL_MODE_NONE;
+    case CullMode::Front:       return VK_CULL_MODE_FRONT_BIT;
+    case CullMode::Back:        return VK_CULL_MODE_BACK_BIT;
+    default:
+        assert(!"Unsupported cull mode");
+        return VK_CULL_MODE_NONE;
+    }
+}
+
+VkFrontFace translateFrontFaceMode(FrontFaceMode frontFaceMode)
+{
+    // TODO: May need to be reversed due to the viewport flip
```

Comment:
That's right, this needs to be reversed.

Reply:
Fixed locally, but doesn't seem to be the cause of nothing being drawn in some of those tests.